argoverse/evaluation/eval_detection.py

```diff
@@ -0,0 +1,329 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+import argparse
+import logging
+import os
+from collections import defaultdict
+from dataclasses import Field, dataclass, field
+from multiprocessing import Pool
+from pathlib import Path
+from typing import DefaultDict, List, Tuple
+
+import matplotlib
+import numpy as np
+import pandas as pd
+from pandas.core import frame
+
+from argoverse.data_loading.object_classes import OBJ_CLASS_MAPPING_DICT
+from argoverse.data_loading.object_label_record import read_label
+from argoverse.evaluation.detection_utils import (
+    DistFnType,
+    FilterMetric,
+    SimFnType,
+    compute_affinity_matrix,
+    dist_fn,
+    filter_instances,
+    get_ranks,
+    interp,
+)
+
+matplotlib.use("Agg")  # isort:skip
+
+
+import matplotlib.pyplot as plt  # isort:skip
+
+
+logger = logging.getLogger(__name__)
+
+
+TP_METRIC_NAMES = ["ATE", "ASE", "AOE"]
+NUM_TP_METRICS = len(TP_METRIC_NAMES)
+
+METRIC_NAMES: List[str] = ["AP"] + TP_METRIC_NAMES + ["CDS"]
+METRIC_DEFAULT_VALUES: List[float] = [0.0, 1.0, 1.0, 1.0, 0.0]
```

Comment:
I might add inline above:
```python
# each metric below is bounded in [0,1], and for ATE, ASE, AOE, lower is better
```

Reply:
explained in module level docstring now, lemme know if you still prefer adding this